### PR TITLE
#167164975 add swagger to project

### PR DESCRIPTION
**Pull request by benisi**
#### Pivotal tracker story
[#167164975](https://www.pivotaltracker.com/story/show/167164975) 
#### What does this PR do?
 Add swagger documentation to project
#### Summary of Task
This PR adds the following to the project
- add swagger.json file to the doc directory
- add documentation URL 
- add swagger-ui-express to package.json
#### How can this be tested?
```
# Clone this repository
$ git clone https://github.com/andela/dahlia-ah-backend.git

# Navigate to the application folder
$ cd dahlia-ah-backend

# Install dependencies
$ npm install

# start the server 

by typing
$ npm run dev:start

# visit http://localhost:3000/docs
```

### CODE REVIEW

File: docs/swagger.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        "password" : {
+          "type" : "string"
+        }
+      },
+      "xml" : {
+        "name" : "User"
+      }
+    }
+  }
+}
```
Comment (allebd) [inline docs/swagger.json]: Kindly add a newline at the end of the file

Comment (benisi) [reply docs/swagger.json]: done

**Comment by OvieMudi** [comment]
The task is well done. But I'm concerned about how the JSON doc being used.
It could really become very complicated to update when each team member starts documenting their various features.

My suggestion: consider using an npm package that automatically converts .yaml to .json format. It's easier to update a YAML file.

Check out this package - https://www.npmjs.com/package/yamljs

**Comment by benisi** [comment]
> The task is well done. But I'm concerned about how the JSON doc being used.
> It could really become very complicated to update when each team member starts documenting their various features.
> 
> My suggestion: consider using an npm package that automatically converts .yaml to .json format. It's easier to update a YAML file.
> 
> Check out this package - https://www.npmjs.com/package/yamljs

I think it is a matter of personal preference, I prefer JSON and I will seek the opinion of other team members to see which one they prefer and implement it

**Comment by OvieMudi** [comment]
> > The task is well done. But I'm concerned about how the JSON doc being used.
> > It could really become very complicated to update when each team member starts documenting their various features.
> > My suggestion: consider using an npm package that automatically converts .yaml to .json format. It's easier to update a YAML file.
> > Check out this package - https://www.npmjs.com/package/yamljs
> 
> I think it is a matter of personal preference, I prefer JSON and I will seek the opinion of other team members to see which one they prefer and implement it

Okay cool. Do seek the opinion of the other team members.

**Review by WilliamsOhworuka — APPROVED** [review]

**Review by Nerocodes — APPROVED** [review]
LGTM

**Review by OvieMudi — APPROVED** [review]
LGTM

**Review by WilliamsOhworuka — APPROVED** [review]

**Pull request merged**